myFirstWebsite/index.html

```diff
@@ -0,0 +1,440 @@
+            <div class="benefits__wrapper">
+                <h2 class="benefits__title common-title" >
+                    Benefits of Odigo
+                </h2>
+                <div class="benefits__items">
+                    <div class="benefits__item">
+                        <div class="benefits__item-icon">
+                            <img src="./images/pagoda-house-icon.png" alt="pagoda house">
+                        </div>
+                        <h2 class="benefits__item-title">
+                            Welcome to Odigo!
+                        </h2>
+                        <p class="benefits__item-description">
+                            Jump off balcony, onto stranger's head. Chase ball of string hide when guests come over.
+                        </p>
+                        <a href="#!" class="benefits__item-more-link common-link">
+                            learn more
+                        </a>
+                    </div>
+                    <div class="benefits__item">
+                        <div class="benefits__item-icon">
+                            <img src="./images/person-icon.png" alt="person">
+                        </div>
+                        <h2 class="benefits__item-title">
+                            Your Personal Japan Guide
+                        </h2>
+                        <p class="benefits__item-description">
+                            Jump off balcony, onto stranger's head. Chase ball of string hide when guests come over.
+                        </p>
+                        <a href="#!" class="benefits__item-more-link common-link">
+                            learn more
+                        </a>
+                    </div>
+                    <div class="benefits__item">
+                        <div class="benefits__item-icon">
+                            <img src="./images/build-icon.png" alt="build">
+                        </div>
+                        <h2 class="benefits__item-title">
+                            Promoting Local Businesses
+                        </h2>
+                        <p class="benefits__item-description">
+                            Jump off balcony, onto stranger's head. Chase ball of string hide when guests come over.
+                        </p>
+                        <a href="#!" class="benefits__item-more-link common-link">
+                            learn more
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="locations" id="locations">
+            <div class="locations__wrapper ">
+                <div class="locations__title-wrapper">
+                    <h2 class="locations__title common-title custom-title-underline" >
+                        Get inspired for your next trip
+                    </h2>
+                    <a href="" class="locations__view-all-link common-link">
```

Comment:
locations__view-all-link looks like you hasn't declared `locations__view-all-link` into CSS file, pls get rid of all unused code.